config/settings/base.py

```diff
@@ -416,9 +416,3 @@
 
 ETHERSCAN_API_KEY = env("ETHERSCAN_API_KEY", default=None)
 IPFS_GATEWAY = env("IPFS_GATEWAY", default="https://cloudflare-ipfs.com/")
-
-SWAGGER_SETTINGS = {
```

Comment:
Maybe we can leave this? It doesn't harm to have it, right?

Reply:
Yes makes sense. I don't think this was being used before on our Swagger.

Reply:
ae8b48a

Reply:
Looks good to me